package.json

```
@@ -7,9 +7,9 @@
     "url": "git+https://github.com/AxaFrance/design-system.git"
   },
   "scripts": {
-    "start": "turbo start start:storybook",
-    "build": "turbo build build:storybook --no-daemon --concurrency 1",
-    "lint": "turbo eslint stylelint prettier tsc --no-daemon",
+    "start": "concurrently \"npm run start --workspace=packages/css\" \"npm run start:storybook --workspaces\"",
+    "build": "npm run build --workspaces && npm run build:storybook --workspaces",
+    "lint": "npm run eslint --workspaces && npm run stylelint --workspace=packages/css && npm run prettier --workspaces && npm run tsc --workspaces",
```

Cette commande est bien mais elle stop à la première erreur du coup je propose

```suggestion
    "lint": "concurrently \"npm:eslint --workspaces\" \"npm:stylelint --workspaces\" \"npm:prettier --workspaces\" \"npm:tsc --workspaces\" -c auto -n eslint,stylelint,prettier,tsc",
```